main.go

```diff
@@ -33,16 +33,37 @@ func recoverPanic() {
 func main() {
 
 	//
-	// Setup our default logging level
+	// Setup our default logging level, which will show
+	// both warnings and errors.
 	//
 	loggerLevel = &slog.LevelVar{}
 	loggerLevel.Set(slog.LevelWarn)
 
 	//
-	// Allow showing "all the logs"
+	// If the user wants a different level they can choose it.
+	//
+	level := os.Getenv("LOG_LEVEL")
+
+	//
+	// Legacy/Compatibility
 	//
 	if os.Getenv("LOG_ALL") != "" {
+		level = "DEBUG"
```

Comment:
Usually, software set ERROR or WARN as default and requests users to manually opt-in for DEBUG. Would having DEBUG as the default be the best option for rss2email users?

Reply:
This is a compatibility thing - in the past "LOG_ALL" used to configure showing debug-messages.  (Could be used in preference to adding `-debug` flag to the run-modes.)

I'd expect "LOG_LEVEL=XXXX" to be the thing the users actually set.

Reply:
oh right, having DEBUG as default will keep compatibility with current behavior. I didn't think at this. Thanks for the explanation.